src/NumericalAlgorithms/Interpolation/InterpolationTargetLineSegment.hpp

```diff
@@ -140,6 +140,31 @@ struct LineSegment {
   using const_global_cache_tags =
       tmpl::list<Tags::LineSegment<InterpolationTargetTag, VolumeDim>>;
   using is_sequential = std::false_type;
+
+  template <typename Metavariables, typename DbTags, typename TemporalId>
+  static tnsr::I<DataVector, VolumeDim, Frame::Inertial> points(
+      const db::DataBox<DbTags>& /*box*/,
+      Parallel::ConstGlobalCache<Metavariables>& cache,
+      const TemporalId& /*temporal_id*/) noexcept {
+    const auto& options =
+        Parallel::get<Tags::LineSegment<InterpolationTargetTag, VolumeDim>>(
```

Comment:
get from the box instead of the cache (all tags in the ConstGlobalCache are available in the DataBox)